bin/deploy-services

```diff
@@ -66,7 +66,9 @@ if [ ! "$LOCAL_DEV" = true ]; then
   envsubst < $file | kubectl apply -f - -n harmony
 fi
 
-export LOCALSTACK_HOST=localhost
+if [ "$current_context" != "colima" ]; then
+  export LOCALSTACK_HOST=localhost
```

Comment:
@ygliuvt - process.env overrides all other sources of environment variables. So I think that if you run this script, and then run the tests, `export LOCALSTACK_HOST=localhost` will override the test values, causing a failure. Try the tests in a new shell -- worked for me.

Reply:
@vinnyinverso This export is in the original script and gets run all the time, and if I do `echo $LOCALSTACK_HOST`, it does not return `localhost` in my current shell.  So I don't think this is causing the failure. It is interesting that it works for you in a new shell, but does not for me.